cla_frontend/settings/base.py

```diff
@@ -153,6 +153,18 @@ def sentry_traces_sampler(sampling_context):
     CSP_FONT_SRC.append(AWS_STORAGE_BUCKET_HOSTNAME)
     CSP_STYLE_SRC.append(AWS_STORAGE_BUCKET_HOSTNAME)
 
+# Security Middleware
+MIDDLEWARE = ("django.middleware.security.SecurityMiddleware", "django.middleware.csrf.CsrfViewMiddleware")
+MIDDLEWARE_CLASSES = ("django_cookies_samesite.middleware.CookiesSameSite",)
```

Comment:
This settings defined further down in the file, you should add your middleware to that list instead.